napari/utils/misc.py

```diff
@@ -149,16 +149,16 @@ def is_iterable(
     provided and the argument is a 1-D array of length 3 or 4 then the input
     is taken to not be iterable. If allow_none is True, `None` is considered iterable.
     """
-    if arg is None and not allow_none:
-        return False
+    if arg is None:
+        return allow_none
     if isinstance(arg, (str, Enum)):
         return False
     if np.isscalar(arg):
         return False
     if color and isinstance(arg, (list, np.ndarray)):
         return np.array(arg).ndim != 1 or len(arg) not in [3, 4]
 
```

Comment:
actually, should we special case affine/np.ndarray in here, like we did with colors?

Reply:
We do not use color anywhere